.rubocop.yml

```diff
@@ -111,11 +111,19 @@ Rails/RedundantActiveRecordAllMethod:
 
 RSpec/FilePath:
   Exclude:
-    - 'spec/data_migrations/*'
+    - 'spec/data_migrations/**/*'
+    - 'spec/models/api/**/*'
+    - 'spec/services/api/**/*'
+    - 'spec/requests/api/**/*'
+    - 'spec/serializers/api/**/*'
 
 RSpec/SpecFilePathFormat:
   Exclude:
-    - 'spec/data_migrations/*'
+    - 'spec/data_migrations/**/*'
+    - 'spec/models/api/**/*'
+    - 'spec/services/api/**/*'
+    - 'spec/requests/api/**/*'
+    - 'spec/serializers/api/**/*'
```

Comment:
Passing thought... :thought_balloon: 

If we was to have it as

```ruby
module V0_1
```
```ruby
module V1_0
``` 
would we need to have the exclusion?